src/main/java/racing/domain/Cars.java

```diff
@@ -40,10 +42,39 @@ public int size() {
     }
 
     public void playRound() {
-        Random random = new Random(System.currentTimeMillis());
+        RandomGenerator randomGenerator = new RandomGenerator(System.currentTimeMillis());
+
+        for (Car car : cars) {
+            car.moveByCondition(randomGenerator.nextInt());
+        }
+    }
+
+    public String getWinnersName(String delimiter) {
+        int maxPosition = maxPosition();
+
+        StringJoiner stringJoiner = new StringJoiner(delimiter);
 
         for (Car car : cars) {
-            car.moveByCondition(random.nextInt(MAX_RANDOM_VALUE));
+            addMaxCarName(maxPosition, stringJoiner, car);
         }
+
+        return stringJoiner.toString();
+    }
+
+    private void addMaxCarName(int maxPosition, StringJoiner stringJoiner, Car car) {
+        if (car.isSamePosition(maxPosition)) {
+            stringJoiner.add(car.getName());
+        }
+    }
+
+    public int maxPosition() {
+        int maxPosition = Integer.MIN_VALUE;
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        int maxPosition = 0;
```

Reply:
무의미 하게 큰 음수값으로 초기화를 하는 것이 좋지 않은 방법인 걸까요?
구글링을 해보려 했는데 어떤 키워드로 구글링 하는게 좋을지 모르겠어서 조언 부탁드립니다 :)

Reply:
A2. 자동차의 DEFAULT_POSITION은 0입니다. 그렇기에 maxPosition()의 최솟값도 DEFAULT_POSITION이 아닐까요? Integer.MIN_VALUE가 자동차 경주 게임에서 어떤 비즈니스적 의미를 지니는지 고민하면 좋겠습니다.

초기값 자체가 도메인 로직과 상관있도록 해야겠네요! 답변 감사합니다! :)